website/docs/d/notification_destination.html.markdown

```diff
@@ -33,18 +33,44 @@ resource "newrelic_notification_channel" "foo-channel" {
 }
 ```
 
+## Name Example Usage
+
+```hcl
+# Data source
+data "newrelic_notification_destination" "foo" {
+  name = "webhook-destination"
+}
+
+# Resource
+resource "newrelic_notification_channel" "foo-channel" {
+  name = "webhook-example"
+  type = "WEBHOOK"
+  destination_id = data.newrelic_notification_destination.foo.id
+  product = "IINT"
+
+  property {
+    key = "payload"
+    value = "{\n\t\"name\": \"foo\"\n}"
+    label = "Payload Template"
+  }
+}
+```
```

Comment:
A small suggestion - can you please run `terraform fmt` on this code snippet and paste it here, just so it is well formatted? We're trying to do this whenever we make new changes to docs, so just wanted to suggest :)

Reply:
Hi @pranav-new-relic , it doesn't look like this command affects this file. do you have another suggestion?

Reply:
Hey @lzaga-newrelic - my bad. I should have been a little more clear - I think there's no straightforward procedure to run this on the file, apart from copying this block to a Terraform file and running `terraform fmt` on it :) I just pushed a commit with the formatting changes 👍

Reply:
thanks :)